<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>tension [0,1]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>grain [5,50]</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tension_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="grain_spinBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QPushButton" name="draw_pushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>绘制样条曲线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空画布</string>
   </property>
  </widget>
  <widget class="QPushButton" name="point_pushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>显示插值点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>速度</string>
   </property>
  </widget>
  <widget class="QSlider" name="speed_horizontalSlider">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>330</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5000</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="sliderPosition">
    <number>7500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="play_pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pause_pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>570</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>By:3180105640 刘佳润</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
